assignment.md

```diff
@@ -1 +1,4 @@
-### Wohoo! We Rock
\ No newline at end of file
+### Wohoo! We Rock
+#### VAISHNAVI GUPTA
+<a href="http://www.github.com/vaishnavi-gupta18"><img src="https://www.flaticon.com/svg/vstatic/svg/2111/2111425.svg?token=exp=1617305974~hmac=990efdb382721ebcc1478147665de63d" width="50px" ></img></a>
```

Comment:
You may choose to remove the github icon or insert the correct icon.
Also, This task must be performed in a single commit. Squash them using interactive rebase and also rebase it with main if needed.